bin/gobblin-mapreduce.sh

```diff
@@ -14,26 +14,47 @@ fi
 
 JOB_TRACKER_URL=$1
 FS_URL=$2
-JOB_CONFIG_FILE=$3
+JOB_CONFIG_FILE=$4
+
+# Convert job jar's relative path to absolute path
+DIRNAME=`dirname "$3"`
+JARNAME=`basename "$3"`
+DIRNAME=`cd "$DIRNAME">/dev/null; pwd`
+JOB_JAR_FILE="$DIRNAME/$JARNAME"
```

Comment:
Just realized that we probably should make this accept a comma-separated list of jar files instead of a single file.